contracts/feature-tests/basic-features/mandos/only_owner.scen.json

```diff
@@ -61,6 +61,86 @@
                 "refund": "*"
             }
         },
+        {
```

Comment:
Can we also move this test to use-module?
I see it was removed from the mandos-rs suite

Reply:
wrote the test initially in basic-features. 
now I placed it in its right place